src/Aop/Pointcut/PointcutParser.php

```diff
@@ -25,4 +26,12 @@ public function __construct(PointcutGrammar $grammar)
         $parseTable = include 'PointcutParseTable.php';
         parent::__construct($grammar, $parseTable);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function parse(TokenStream $stream)
+    {
+        return new AndPointcut(parent::parse($stream), new ClassDeclaresPointcut());
```

Comment:
Ok, seams wrong -> I should only wrap original Pointcut if `getKind()` is PROPERTY or METHOD, right?